crime-commons-samples/build.gradle

```diff
@@ -1,5 +1,5 @@
 plugins {
-    id "org.springframework.boot" version "3.4.0"
+    id "org.springframework.boot" version "3.4.4"
```

Comment:
Shall we refer version from gradlew.properties something like  "org.springframework.boot": springframeworkBootVersion

Reply:
Yeh! Good call I've update to pull the version from the gradle properties file now 👍